cva6/docs/VerifPlans/csr_access/VP_IP004.yml

```diff
@@ -0,0 +1,122 @@
+!Feature
+next_elt_id: 4
+name: CVA6_Machine_mode_counter_csr(mcycle,mcycleh,minstret,minstreth)
```

Comment:
The same comments are per the `CVA6_User_Mode_Counter_CSRs(cycle, instret, cycleh, instreth)` CSRs will apply here.

Reply:
Modified machine Machine mode counters.